gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreTest.java

```diff
@@ -431,7 +431,7 @@ public void testRunStructuredQuery() {
 
     StructuredQuery<ProjectionEntity> projectionQuery = Query.projectionEntityQueryBuilder()
         .kind(KIND2)
-        .projection(Projection.property("age"), Projection.first("name"))
+        .projection(Projection.property("age"))
         .filter(PropertyFilter.gt("age", 18))
         .groupBy("age")
```

Comment:
I think we should be able to do a distinct on one column, so having Projection.property("name") 
and  keeping assertEquals("Dan", entity.getString("name")); should work, no?